consensus/poet/sgx/sawtooth_poet_sgx/libpoet_enclave/poet_enclave.cpp

```diff
@@ -392,6 +394,13 @@ poet_err_t ecall_CreateSignupData(
             ret,
             "Failed to create PoET public/private key pair");
 
+        // Create the monotonic counter bound to the keypair
+        ret =
+            sgx_create_monotonic_counter(
+                &validatorSignupData.counterId,
+                &initialCounterValue);
+        sp::ThrowSgxError(ret, "Failed to create monotonic counter.");
```

Comment:
After tugging at the thread I have unraveled a little further than I originally intended to. I'm now going to get rid of poet_State entirely. I'll update the call pattern such that calls to create the Timer and the WaitCert both need the signupdata. SignupData will effectively become the enclave state.